inf-clojure.el

```diff
@@ -1002,17 +1013,16 @@ the `inf-clojure-prompt`."
       ;; Collect the output
       (set-buffer work-buffer)
       (goto-char (point-min))
-      (let* ((beg (or (when (and beg-string (search-forward beg-string nil t))
-                        (match-beginning 0))
-                      (point-min)))
-             (end (or (when end-string
-                        (search-forward end-string nil t))
-                      (point-max)))
-             (prompt (when (search-forward inf-clojure-prompt nil t)
-                       (match-beginning 0)))
-             (buffer-string (buffer-substring-no-properties beg (or prompt end))))
-        (inf-clojure--log-string buffer-string "<-RES----")
-        buffer-string))))
+      (let* ((buffer-string (buffer-substring-no-properties (point-min) (point-max)))
+             (boundaries (inf-clojure--string-boundaries buffer-string inf-clojure-prompt beg-regexp end-regexp))
+             (beg-pos (car boundaries))
+             (end-pos (car (cdr boundaries)))
+             (prompt-pos (car (cdr (cdr boundaries))))
+             (response-string (substring buffer-string beg-pos (min end-pos prompt-pos))))
+        (inf-clojure--log-string buffer-string "<-BUF----")
```

Comment:
I have tried with `edebug` to debug this kind of process output. No success. Maybe I was doing something wrong. In any case I can take it away. That is the core of the plugin and you need it like bread.